draft-thomson-escape-report.md

```diff
@@ -550,16 +572,16 @@ enabling that technology.
 
 While it isn't clear what all of the outcomes of adopting Web Packaging would
 be, the Workshop revealed several concerns for consolidation risks for all
-involved parties: linking sites, publisher sites, and users.
+involved parties: users, publisher sites, linking sites, and services they each
+rely on.
 
 
 ### Consolidation of Power in Linking Sites
 
-Several participants noted that Web Packaging's enablement of instant
-navigation ({{nav}}) might advantage larger linking sites – especially
-aggregators like social networks and search engines – because doing so requires
-careful selections of which links to optimize, so as not to create unneeded
-traffic.
+Several participants noted that Web Packaging's enablement of instant navigation
+({{nav}}) might advantage larger linking sites over smaller ones in the same
+industry because doing so requires careful selections of which links to
+optimize, so as not to create unneeded traffic.
```

Comment:
Why are we not mentioning social networks and search engines here? They are good examples -- really, the predominant ones.